src/app/test_executive/payments_test.ml

```diff
@@ -441,11 +443,29 @@ module Make (Inputs : Intf.Test.Inputs_intf) = struct
 
             2 successful txn are sent in the prior course of this test,
             so spamming out at least 10 more here will trigger a ledger proof to be emitted *)
-         repeat_seq ~n:10 ~f:(fun () ->
-             Network.Node.must_send_payment ~logger sender ~sender_pub_key
-               ~receiver_pub_key ~amount:Currency.Amount.one ~fee
-             >>| ignore)
+           repeat_seq ~n:10 ~f:(fun () ->
+               Network.Node.must_send_payment ~logger sender ~sender_pub_key
+                 ~receiver_pub_key ~amount:Currency.Amount.one ~fee
+               >>| ignore)
+         in
+         wait_for t
+           (Wait_condition.ledger_proofs_emitted_since_genesis ~num_proofs:1))
+    in
+    section "Running replayer"
+      (let%bind logs =
+         Network.Node.run_replayer ~logger
```

Comment:
would this capture any exceptions that may cause a replayer crash?

Reply:
It's running `kubectl`, and I believe that will raise if the exit code is non-zero.

Reply:
Just verified that the `exec`'d command's exit code is returned from `kubectl`.